### PR TITLE
Standardise DIAMOND output channels and md5sum

### PR DESCRIPTION
**Pull request by jfy133**
Apparently there is a occasional variability causing CI test failure. Removes MD5sum to make this more robust.

## PR checklist

Closes #XXX 

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [ ] Remove all TODO statements.
- [ ] Emit the `versions.yml` file.
- [ ] Follow the naming conventions.
- [ ] Follow the parameters requirements.
- [ ] Follow the input/output options guidelines.
- [ ] Add a resource `label`
- [ ] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - [ ] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`

### CODE REVIEW

File: modules/diamond/blastp/main.nf [hunk 1]
```diff
@@ -11,17 +11,25 @@ process DIAMOND_BLASTP {
     tuple val(meta), path(fasta)
     path db
     val outext
```
Comment (mahesh-panchal) [inline modules/diamond/blastp/main.nf:13]:
```suggestion
    val out_ext
```
Can we change the name of this one too please for readability?

File: modules/diamond/blastp/main.nf [hunk 2]
```diff
@@ -10,19 +10,27 @@ process DIAMOND_BLASTP {
...
 
     when:
     task.ext.when == null || task.ext.when
 
     script:
     def args = task.ext.args ?: ''
     def prefix = task.ext.prefix ?: "${meta.id}"
-    switch ( outext ) {
+    def columns = blast_columns ? "${blast_columns}" : ''
+    switch ( out_ext ) {
```
Comment (mahesh-panchal) [inline modules/diamond/blastp/main.nf:33]: Can we add a default case with a warning if the `out_ext` is unrecognised?

File: tests/modules/diamond/blastx/main.nf [hunk 3]
```diff
@@ -9,18 +9,20 @@ workflow test_diamond_blastx {
 
     db = [ file(params.test_data['sarscov2']['genome']['proteome_fasta'], checkIfExists: true) ]
     fasta = [ file(params.test_data['sarscov2']['genome']['transcriptome_fasta'], checkIfExists: true) ]
-    outext = 'txt'
+    out_ext = 'tfdfdt'
```
Comment (mahesh-panchal) [inline tests/modules/diamond/blastx/main.nf:12]:
```suggestion
    out_ext = 'tfdfdt'  // Nonsense file extension to check default case.
```

**Review by mahesh-panchal — APPROVED** [review]
LGTM. Thanks for getting this sorted.

File: tests/modules/diamond/blastx/main.nf (hunk 3 above)
Comment (jfy133) [reply tests/modules/diamond/blastx/main.nf:12]: Ah yes, forgot to revert that :laughing: , keep it in makes sense anywhay :+1:

**Pull request merged**